dockers/docker-base-buster/sources.list

```diff
@@ -6,3 +6,5 @@ deb-src [arch=amd64] http://debian-archive.trafficmanager.net/debian/ buster mai
 deb [arch=amd64] http://debian-archive.trafficmanager.net/debian-security/ buster/updates main contrib non-free
 deb-src [arch=amd64] http://debian-archive.trafficmanager.net/debian-security/ buster/updates main contrib non-free
 deb [arch=amd64] http://debian-archive.trafficmanager.net/debian/ buster-backports main contrib non-free
+deb [arch=amd64] http://packages.trafficmanager.net/debian/debian buster main contrib non-free
```

Comment:
>packages.trafficmanager.net/debian/debian [](start = 24, length = 41)

Could you add some comments about [http://packages.trafficmanager.net/debian/debian](http://packages.trafficmanager.net/debian/debian)?
Seem it is a special mirror you created.

Reply:
Add comments